packages/components/src/components/modal/modal.js

```diff
@@ -136,6 +136,9 @@ class Modal extends mixin(
         this.options.classBody,
         false
       );
+      // Trying to add in the ability to move the focus back to the trigger or a dev-defined element but I don't know
```

Comment:
As you can see, the "focus-pop" behavior hasn't been implemented yet. I think the approach will be keep a ref to `this.element.ownerDocument.activeElement` right before we move focus upon opening modal.